providers/openstack/README.md

```diff
@@ -94,43 +101,67 @@ clouds:
 ```
 
 ```bash
-helm upgrade -i csp-helper-my-tenant -n my-tenant --create-namespace https://github.com/SovereignCloudStack/openstack-csp-helper/releases/download/v0.3.0/v0.3.0.tgz -f path/to/clouds.yaml
+CSP_HELPER_VERSION="$(curl https://api.github.com/repos/SovereignCloudStack/openstack-csp-helper/releases/latest -s | jq .name -r)"
+helm upgrade -i csp-helper-"${CS_NAMESPACE}" -n "${CS_NAMESPACE}" --create-namespace https://github.com/SovereignCloudStack/openstack-csp-helper/releases/download/"${CSP_HELPER_VERSION}"/openstack-csp-helper.tgz -f path/to/clouds.yaml
 ```
 
 ## Create Cluster Stack definition (CSP/per tenant)
 
+Configure the Cluster Stack you want to use:
+
+```sh
+# the name of the cluster stack (must match a name in https://github.com/SovereignCloudStack/cluster-stacks/tree/main/providers/openstack
+export CS_STACK_NAME=alpha
+
+# the kubernetes version of the cluster stack (must match a tag for the kubernetes version and the stack version)
+export CS_KUBERNETES_VERSION=1.29
```

Comment:
Maybe using `CS_K8S_VERSION` is better to shorten the name of the variable, but this is optional.